js/index.js

```diff
@@ -0,0 +1,189 @@
+// Declaration of variables
+const body = document.querySelector("#body");
+const hamburger = document.querySelector(".hamburger");
+const hamburgerPopUpNav = document.querySelectorAll(".pop-up-ul-text");
+
+// Footer of page
+// Store Artist information in object
+
+const featuredArtists = [
+  {
+    id: `1`,
+    artistImage: `./images/haywaya.jpg`,
+    artistName: `Haywaya`,
+    imageAlt: `Picture of Haywaya`,
+    artistLabel: `Hybrid Rekords recording artist.`,
+    latestRelease: `Haywaya just release a new EP titled "All That I Am".`,
+  },
+
+  {
+    id: `2`,
+    artistImage: `./images/drvmroll.jpg`,
+    artistName: `Drvmroll`,
+    imageAlt: `Picture of Drvmroll`,
+    artistLabel: `Independent recording artist.`,
+    latestRelease: `Drvmroll has produced for A-list artist like Sarkodie.`,
```

Comment:
Great job @Haywayaheadshot 🤝

- [ ] Kindly make sure your design mimics the template provided as much as possible. You may display your featured speakers on two columns instead of one on the desktop version of the home page.
- Your design
![your speakers](https://user-images.githubusercontent.com/63917725/191541621-8f6bf114-2858-43fb-ae07-9b01c5a794d4.PNG)

- Given design
![speakers desktop](https://user-images.githubusercontent.com/63917725/191541698-2aaa7b07-db87-4d3a-ab41-7ef52a4fd6fd.PNG)

Reply:
@1GORDON Thank you for reviewing my project. I have done all the changes but one. 

I am having difficulties achieving the exact same display due to the fact that elements were created with javascript. If you have the time, please do help me out.